src/dawn/IIR/AST.h

```diff
@@ -0,0 +1,26 @@
+//===--------------------------------------------------------------------------------*- C++ -*-===//
```

Comment:
What is the point oif the IIR-AST header?

Reply:
IIR/, SIR/ and AST/ header files of the kind `AST*.h` all have the same mirrored structure.
If some piece of code that works with IIR was including, before the PR, "SIR/AST.h" now it includes "IIR/AST.h". Always including ASTFwd.h may be bad for compilation time, as, to provide aliases, ASTFwd.h includes a lot of other headers.